tests/README.md

```diff
@@ -0,0 +1,74 @@
+# Test documentation
+
+## List of scripted integration tests
+
+1. Up
+    * file: up.bash
+    * purpose: test if the SonarQube server is UP
+1. Plugin check
+    * file: check_plugins.bash
+    * purpose: check that the plugins listed in the README are installed on the server with the right version
+1. Quality Gate check
+    * file: check_qg.bash
+    * purpose: check that the CNES Quality Gate is available on the server and is set as default
+1. Quality Profiles check
+    * file: check_qp.bash
+    * purpose: check that all CNES Quality Profiles are available on the server
+
+## How to run all the tests
+
+Before testing the image, it must be built with `docker build -t lequal/docker-cat .`.
+
+To run the tests, the following tools are required:
+
+* `curl`
+* `jq`
+
+To run all the tests, use the test script:
+
+```sh
+# from the root of the project
+$ ./tests/run_tests.bash
+```
+
+## How to run a specific test
+
+1. Run a container of the image (see the [user guide](https://github.com/cnescatlab/docker-cat#Quick-install))
+1. Wait until it is configured
+    * The message `[INFO] Docker CAT is ready to go and find bugs!` is logged.
+1. Run a script
+    ```sh
+    $ ./tests/up.bash
+    ```
+    * Environnement variables may be modified
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/cnescatlab/docker-cat/pullRequest?prid=6196008)